viewer/src/data/worker/WorkerPool.ts

```diff
@@ -16,7 +16,15 @@ interface PooledWorker {
 }
 
 export class WorkerPool {
+  static get defaultPool(): WorkerPool {
```

Comment:
Unrelated change?

Reply:
I like it, though :)

Reply:
Kinda unrelated, but its to support multiple models better by not flooding with N workers for each model added.